packages/report-schema/package.json

```diff
@@ -28,5 +30,8 @@
   "bugs": {
     "url": "https://github.com/stryker-mutator/mutation-testing-elements/issues"
   },
-  "homepage": "https://github.com/stryker-mutator/mutation-testing-elements/tree/master/packages/elements#readme"
+  "homepage": "https://github.com/stryker-mutator/mutation-testing-elements/tree/master/packages/elements#readme",
+  "devDependencies": {
+    "ajv": "^8.6.3"
```

Comment:
Why is this lone `devDependency` needed here, instead of hoisting that to the top as well?

Reply:
I've moved packages that are only used by a single module to that module. IMO it makes more sense that, for example, `lit` is a devDependency in elements instead of in root. Since report-schema is the only one that uses `ajv` it belongs in here.

I _think_ this should also give more finegrained behaviour when `nx` updates this package (it'll only run tasks for this package and its dependencies, instead of all of them)